controller/handler/trial-csv.js

```diff
@@ -53,6 +43,16 @@ const configuration = [
         label: 'question option',
         key: 'optionText',
         default: ''
+    },
+    {
```

Comment:
This is moving the start and end away from the patient information.
It should be next to the patient pin number.